sycl/doc/GetStartedGuide.md

```diff
@@ -145,6 +150,24 @@ above.
 
 # Use DPC++ toolchain
 
+## Using the SYCL toolchain on CUDA platforms
+
+The SYCL toolchain support on CUDA platforms is still in an experimental phase.
```

Comment:
```suggestion
The DPCPP toolchain support on CUDA platforms is still in an experimental phase.
```

Reply:
Please, replace other SYCL usages with DPCPP

Reply:
DPC++ looks better!

Reply:
SYCL DPC++ looks better :-)

Reply:
> SYCL DPC++ looks better :-)

Bit of a repetition there, right? ;-)